templates/focus-area-styles.css

```diff
@@ -1,12 +1,17 @@
 body {
-    background-color: rgb(246,247,249);
+    background-color: rgb(247,247,247);
+    padding-left: 20px;
+    padding-right: 20px;
 }
 
 
 #focus-area-wrapper {
-  border: 1px solid #D0D0D0;
-  border-top: 4px solid rgb(74,164,74);
-  box-shadow: 0 0 0 1px rgba(0, 0, 0, .1), 0 2px 3px rgba(0, 0, 0, .2);
-  padding: 1.5em;
+  /*border: 1px solid #D0D0D0;*/
+  /*border-top: 4px solid rgb(74,164,74);*/
+  /*box-shadow: 0 0 0 1px rgba(0, 0, 0, .1), 0 2px 3px rgba(0, 0, 0, .2);*/
+  border-radius: 10px;
+  border-bottom: 1px solid rgb(204, 204, 204);
+  padding: 2em;
+  padding-bottom: 3em;
```

Comment:
IMO, dropping the green top border makes the page significantly less friendly.

Reply:
Josh, I agree the green border was nice and the revised color scheme is a bit gray. I've been experimenting with some further changes that fix these short comings. They are changes on top of this PR, so will make these first.